### PR TITLE
Use cards on template hub

### PR DESCRIPTION
**Pull request by Arthurlpgc**

### CODE REVIEW

File: vitaes-webapp/src/TemplateHub/TemplateHubModel/index.js [hunk 1]
```diff
@@ -22,62 +23,64 @@ export const TemplateHubModel = (props) => {
...
+            src={model.data.image}
+            onClick={() => setShow(true)}
+          />
+          <Card.Content style={{ background: '#f5f5f5' }}>
+            <Card.Header>{capitalize(props.keyName)}</Card.Header>
+
+            <div
+              className="template-button"
+              role="button"
+              onClick={() => {
```
Comment (RamonSaboya) [inline vitaes-webapp/src/TemplateHub/TemplateHubModel/index.js]: Shouldn’t this be a class function?

Comment (Arthurlpgc) [reply vitaes-webapp/src/TemplateHub/TemplateHubModel/index.js]: Fair enough, cant escape from 2 code reviews can I? haha

Comment (RamonSaboya) [reply vitaes-webapp/src/TemplateHub/TemplateHubModel/index.js]: Never hahah

Comment (Arthurlpgc) [reply vitaes-webapp/src/TemplateHub/TemplateHubModel/index.js]: Ooops
fixeed wrong part

Comment (Arthurlpgc) [reply vitaes-webapp/src/TemplateHub/TemplateHubModel/index.js]: It will be a function, not a class function

**Pull request merged**